src/core/currency/wallet/currency-wallet-files.ts

```diff
@@ -31,31 +32,38 @@ import {
 } from './currency-wallet-cleaners'
 import { CurrencyWalletInput } from './currency-wallet-pixie'
 import { TxFileNames } from './currency-wallet-reducer'
+import { currencyCodesToTokenIds } from './enabled-tokens'
 
 const CURRENCY_FILE = 'Currency.json'
-const ENABLED_TOKENS_FILE = 'EnabledTokens.json'
 const LEGACY_MAP_FILE = 'fixedLegacyFileNames.json'
+const LEGACY_TOKENS_FILE = 'EnabledTokens.json'
+const TOKENS_FILE = 'Tokens.json'
 const WALLET_NAME_FILE = 'WalletName.json'
 
-const enabledTokensFile = makeJsonFile(asEnabledTokensFile)
 const legacyAddressFile = makeJsonFile(asLegacyAddressFile)
 const legacyMapFile = makeJsonFile(asLegacyMapFile)
+const legacyTokensFile = makeJsonFile(asLegacyTokensFile)
 const legacyTransactionFile = makeJsonFile(asLegacyTransactionFile)
+const tokensFile = makeJsonFile(asTokensFile)
 const transactionFile = makeJsonFile(asTransactionFile)
 const walletFiatFile = makeJsonFile(asWalletFiatFile)
 const walletNameFile = makeJsonFile(asWalletNameFile)
 
 /**
  * Updates the enabled tokens on a wallet.
  */
-export async function changeEnabledTokens(
+export async function writeTokensFile(
```

Comment:
Mark as deprecated

Reply:
Fixed: We were using the wrong filename. It should be saving to `TOKENS_FILE`.